client/src/components/layout/DestinationShow.js

```diff
@@ -27,14 +29,19 @@ const DestinationShow = (props) => {
   }, []);
 
   return (
-    <div className="destination">
-      <h1>{name}</h1>
-      <p>{description}</p>
-      <ul>
-        <li>{type}</li>
-        <li>{location}</li>
-        <li>{website}</li>
-      </ul>
+    <div>  
```

Comment:
empty divs should be replaced with react fragments `<></>` unless you are applying a classname to them